onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -28,102 +231,53 @@
         )
 
 
-def quantize_static(
-    model_input,
-    model_output,
-    calibration_data_reader: CalibrationDataReader,
-    quant_format=QuantFormat.QDQ,
-    op_types_to_quantize=[],
-    per_channel=False,
-    reduce_range=False,
-    activation_type=QuantType.QInt8,
-    weight_type=QuantType.QInt8,
-    nodes_to_quantize=[],
-    nodes_to_exclude=[],
-    optimize_model=True,
-    use_external_data_format=False,
-    calibrate_method=CalibrationMethod.MinMax,
-    extra_options={},
-):
-
-    """
-        Given an onnx model and calibration data reader, create a quantized onnx model and save it into a file
+def quantize_static(model_input, model_output, calibration_data_reader: CalibrationDataReader, **kwargs):
```

Comment:
>quantize_static [](http://example.com/codeflow?start=4&length=15)

I'm thinking this may be a little inconvenient for user to figure out how to call the APIs. I think it is better to add an extra API
def quantize(model_input, model_output, quant_config: QuantConfig)

And StaticQuantConfig and DynamicQuantConfig derive from QuantConfig.

And then quantize call either quantize_static or quantize_dynamic based on the real type of QuantConfig